tools/ci_build/github/azure-pipelines/orttraining-linux-gpu-frontend-test-ci-pipeline.yml

```diff
@@ -1,42 +1,257 @@
-# 'trigger: none' to use UI to schedule the pipeline runs.
-# alternatively to schedule the pipeline run according to:
-# https://docs.microsoft.com/en-us/azure/devops/pipelines/process/scheduled-triggers?view=azure-devops&tabs=yaml
 trigger: none
 
 jobs:
-- job: Onnxruntime_Linux_GPU_Training_FrontEnd
+- job: Onnxruntime_Linux_GPU_Training_E2E_Test_Nightly
 
-  timeoutInMinutes: 300
+  timeoutInMinutes: 120
 
   steps:
   - checkout: self
     clean: true
     submodules: recursive
 
-  - template: templates/set-test-data-variables-step.yml
-
-  - task: CmdLine@2
-    displayName: 'Clean untagged docker images'
-    inputs:
-      script: |
-        docker rm $(docker ps -a | grep Exited | awk '{print $1;}') || true
-        docker container prune -f
-        docker image prune -f
-      workingDirectory: $(Build.BinariesDirectory)
-    continueOnError: true
-    condition: always()
-
-  - script: >
-      tools/ci_build/github/linux/run_dockerbuild.sh
-      -o ubuntu16.04 -d gpu -r $(Build.BinariesDirectory)
-      -x "
-      --enable_training
-      --config RelWithDebInfo
-      --skip_onnx_tests
-      --build_wheel
-      --enable_training_python_frontend_e2e_tests
-      --enable_training_pipeline_e2e_tests
-      "
-    displayName: 'Build and run frontend tests'
-
-  - template: templates/clean-agent-build-directory-step.yml
\ No newline at end of file
+  # update these if the E2E test data changes
+  - script: |
+      orttraining/tools/ci_test/download_e2e_test_data.py \
```

Comment:
nit: can we rename this to download_azure_blob.py or something more general?

Reply:
name changed